src/main/java/ru/sbt/mipt/oop/SmartHomeHandler.java

```diff
@@ -0,0 +1,31 @@
+package ru.sbt.mipt.oop;
+
+import ru.sbt.mipt.oop.home_components.SmartHome;
+import ru.sbt.mipt.oop.processors.Processor;
+
+import java.util.List;
+
+public class SmartHomeHandler {
+    private final SmartHome smartHome;
+    private final List<Processor> processors;
+    private SensorEvent event;
+
+    {
+        event = RandomDataGenerator.getNextSensorEvent();
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%BF%D0%B5%D1%80%D0%B5%D0%B4%D0%B0%D0%B2%D0%B0%D0%B9%D1%82%D0%B5-%D0%B7%D0%B0%D0%B2%D0%B8%D1%81%D0%B8%D0%BC%D0%BE%D1%81%D1%82%D0%B8-%D1%81%D0%BD%D0%B0%D1%80%D1%83%D0%B6%D0%B8